net.lutris.Lutris.yml

```diff
@@ -1,7 +1,7 @@
 id: net.lutris.Lutris
-sdk: org.freedesktop.Sdk
-runtime: org.freedesktop.Platform
-runtime-version: "19.08beta"
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: master
```

Comment:
Please change `master` to the some branched version once it's available to install.

Reply:
Changed to 3.34beta